pages/common/airpaste.md

```diff
@@ -0,0 +1,15 @@
+# airpaste
+
+> Share messages and files locally.
```

Comment:
"locally" might mean on the local machine. I think we should say "local network" for clarity.